specification/automation/resource-manager/Microsoft.Automation/preview/2022-06-30-preview/runtime.json

```diff
@@ -0,0 +1,422 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AutomationManagement",
+    "version": "2022-06-30-preview",
+    "x-ms-code-generation-settings": {
+      "useDateTimeOffset": true
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}/runtimes/{runtimeName}": {
+      "delete": {
+        "tags": [
+          "Runtime"
+        ],
+        "operationId": "Runtime_Delete",
+        "description": "Delete a runtime.",
+        "externalDocs": {
+          "url": "http://aka.ms/azureautomationsdk/runtimeconfigurationoperations"
+        },
+        "x-ms-examples": {
+          "Delete a runtime": {
+            "$ref": "./examples/deleteRuntime.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
```

Comment:
>ResourceGroupNameParameter [](http://example.com/codeflow?start=66&length=26)

Any particular reason you need to create ResourceGroupNameParameter under your swagger instead of using v3 common types? Same for the other common parameters

Reply:
I had tried changes for using v3 common types for ResourceGroupNameParameter, SubscriptionIdParameter and ApiVersionParameter. Here is the commit - https://github.com/Azure/azure-rest-api-specs/pull/20357/commits/d512f81a36871a16faedeeaff75f0da3ec84e4cb

But it is introducing breaking changes - so reverted back this commit.

<img width="931" alt="image" src="https://user-images.githubusercontent.com/50352900/187247078-cbee5fe2-11b7-4fc3-8358-e977b5e0e0f2.png">